doc/devel/coding_guide.rst

```diff
@@ -439,21 +441,30 @@ Other branches are fed through :ref:`automatic <automated-backports>` or
 targeting other branches is only rarely necessary for special maintenance
 work.
 
-.. backport_strategy:
+.. _backport-strategy:
 
 Backport strategy
 -----------------
 
-We will always backport to the patch release branch (*v3.N.x*):
+Backports to the patch release branch (*v3.N.x*) are the changes that will be
+included in the next patch (aka bug-fix) release.  The goal of the patch
+releases is to fix bugs introduced in the associated minor release without
```

Comment:
```suggestion
releases is to fix bugs without
```

The goal is to fix bugs (not limited) to when they were introduced.

The decision on whether an older bug goes into a patch release is the result of the trade off between “fix” and “without adding any new regressions or behavior changes”.